### ISSUE TITLE
Strange Behaviour for FritzRepeater

### ISSUE THREAD

**Issue by Laeae** [SeydX/homebridge-fritz-platform#139]
Hi, 

I have setup my FritzRepeater with a switch for the repeater itself and a switch for the wireless LAN. 

When viewing these switches in Home, the switches turn on and off by themselves, without my interfering. This is very strange behaviour and not as expected. For example I cannot turn back on the repeater, after I have turned it off. 

Please also see the video, that shows this behaviour:
https://ufile.io/n1t858fs (this is an .mp4 file on a free sharing service, no strings nor viruses attached)

Thanks,
Laeae

**Comment by SeydX** [comment]
Hi @Laeae 

yeah thats really strange, can u post your config.json pls

**Comment by Laeae** [comment]
It does not look too strange I would say... 


```
{
	"platform": "FritzPlatform", 
	"name": "Fritzbox Platform",
            "devices": [
                {
                    "name": "XXX",
                    "host": "192.168.178.1",
                    "username": "XXX",
                    "password": "XXX",
                    "ssl": true,
                    "tr064": true,
                    "igd": true,
                    "master": true,
                    "hide": true,
                    "connection": "dsl",
                    "options": {
                        "wifi_2ghz": "switch",
                        "wifi_guest": "switch",
                        "wps": "switch",
                        "deflection": "switch"
                    }
                },
                {
                    "name": "Fritz Repeater",
                    "host": "192.168.178.54",
                    "username": "XXX",
                    "password": "XXX",
                    "ssl": true,
                    "tr064": true,
                    "igd": false,
                    "master": false,
                    "connection": "repeater",
                    "options": {
                        "wifi_2ghz": "switch"
                    }
                }
            ],
            "presence": [
                {
                    "active": true,
                    "blockTelegram": true,
                    "name": "XXX",
                    "address": "192.168.178.64",
                    "accType": "occupancy",
                    "ping": false
                },
                {
                    "active": true,
                    "blockTelegram": true,
                    "name": "XXX",
                    "address": "192.168.178.25",
                    "accType": "occupancy",
                    "ping": false
                }
            ],
            "smarthome": [
                {
                    "name": "Küchenheizung",
                    "ain": "133560211864",
                    "temperature": false,
                    "battery": false,
                    "accType": "thermostat"
                },
                {
                    "name": "Wohnzimmerheizung links",
                    "ain": "119630448512",
                    "temperature": false,
                    "battery": false,
                    "accType": "thermostat"
                },
                {
                    "name": "Wohnzimmerheizung rechts",
                    "ain": "119600990256",
                    "temperature": false,
                    "battery": false,
                    "accType": "thermostat"
                },
                {
                    "name": "Flurheizung",
                    "ain": "099950485858",
                    "temperature": false,
                    "battery": false,
                    "accType": "thermostat"
                },
                {
                    "name": "Badheizung",
                    "ain": "099950485859",
                    "temperature": false,
                    "battery": false,
                    "accType": "thermostat"
                },
                {
                    "name": "Heizung im Arbeitszimmer",
                    "ain": "133570300536",
                    "temperature": false,
                    "battery": false,
                    "accType": "thermostat"
                },
                {
                    "name": "Außenlicht",
                    "ain": "116300087273",
                    "energy": false,
                    "temperature": true,
                    "accType": "switch"
                }
            ],
            "callmonitor": {
                "active": false,
                "port": 1012
            },
            "extras": {
                "alarm": {
                    "active": false
                },
                "wakeup": {
                    "active": false
                },
                "ringlock": {
                    "active": false
                },
                "phoneBook": {
                    "active": false
                }
            },
            "options": {
                "polling": {
                    "timer": 10
                }
            },
            "telegram": {
                "active": true,
                "token": "XXX",
                "chatID": "XXX"
            }
        },
```

**Comment by Laeae** [comment]
The repeater is part of the Mesh network, and thus has the same login credentials as the Fritzbox.

**Comment by SeydX** [comment]
Does the problem only appear sometimes? Or is it there all the time? Could you switch on "debug" and then send the log to

**Comment by Laeae** [comment]
I'm sorry, due to lack of time I have disabled the device again. Once my life becomes a little less stressed, I will happily uptake the testing again.

**Comment by SeydX** [comment]
Fixed in v5.0.25
